transmogrifier/sources/transformer.py

```diff
@@ -55,73 +68,246 @@ def __next__(self) -> TimdexRecord:
                 self.skipped_record_count += 1
                 continue
 
+    @final
+    @staticmethod
+    def get_transformer(source: str) -> Transformer:
```

Comment:
Apologies, think I misspoke in our chat, think this return type should be a `type[Transformer]` to indicate it's not yet initialized, e.g.:

```python
def get_transformer(source: str) -> type[Transformer]:
...
```

This resolves an IDE linting error in `cli.main` where an instance is not callable.